__tests__/app.test.js

```diff
@@ -123,7 +123,46 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("GET 200 responds with an array of the articles with the requested topic.", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(1);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
```

Comment:
We should hard code the topic property here as every article should have the topic 'cats' if we've queried it correctly.